mine_extractor_refactored/config/settings.py

```
@@ -0,0 +1,111 @@
+# ... existing code ...
+
+def get_config(config_file: str = "default.json") -> Dict[str, Any]:
```

Ruff (0.12.2)

Undefined name `Dict`

(F821)